addons/mrp/models/mrp_workcenter.py

```diff
@@ -610,7 +610,6 @@ class MrpWorkcenterCapacity(models.Model):
     _name = 'mrp.workcenter.capacity'
     _description = 'Work Center Capacity'
     _check_company_auto = True
-    _order = 'workcenter_id, product_id nulls first,product_uom_id,id'
```

Comment:
Why this change? Nothing in the task nor the commit description justifies it 🤔

Reply:
oups, fixed : commit & pr messages updated